pkg/interfaces/contracts/vault/VaultTypes.sol

```diff
@@ -162,6 +166,16 @@ struct TokenInfo {
     bool paysYieldFees;
 }
 
+/**
+ * @dev Data structure used to represent the current pool state in memory
+ * @param asdf
+ * @param
+ * @param
+ * @param
+ * @param balancesLiveScaled18 Token balances after paying yield fees, applying decimal scaling and rates
+ * @param
+ * @param
```

Comment:
I guess this was not intended? hahahaha

Reply:
Ha! Didn't finish that one I guess. Thanks!